webxrlayers-1.bs

```diff
@@ -1875,6 +1875,18 @@ when the [=XR Compositor=] can no longer reproject the layer.
 
 The author SHOULD redraw the content of the layer at the next [=XR animation frame=]. The event must be of type {{XRLayerEvent}}.
 
+Depth sorting between layers {#depthsorting}
+============================
+By default, the {{XRRenderStateInit/layers}} array defines the order of the composition of the layers and each layer is drawn on top of the previous layer.
+If an application wants to have layers that are sorted by depth, it MUST request a session with the "<dfn for="feature descriptor">depth-sort</dfn>" [=feature descriptor=].
+
+If an {{XRSession}} is created with the "[=depth-sort=]" [=feature descriptor=], {{XRProjectionLayer}}, {{XRQuadLayer}} and {{XRCylinderLayer}} layers MUST be displayed based on their depth as opposed to the location in the {{XRRenderStateInit/layers}} array.
```

Comment:
I would prefer that the feature descriptor contain the term "layer" somewhere, because otherwise "depth-sort" feels a bit too generic.

"depth-sort-layers" or "layer-depth-sorting" perhaps?